source/reporting/Report database.rst

```diff
@@ -0,0 +1,248 @@
+--------------------------------
+Information about payment requirements, connected to the subscription (Subscription) and what is the next subscriptionId.
+
+Payment claims are formed before the subscription is formed. Therefore, subscriptionid is for the previous subscription, while NextsubscriptionId is the subscription in question. When a subscription is renewed, NextSubscriptionId is applied to the correct payment claim.
+ 
+[economy].[PaymentDemandFees]
+---------------------------
+Invoice fee.
+
+
+[economy].[PaymentDemands]
+--------------------------
+Invoicing of customers produces a demand for payment, and PaymentDemands contains customers' invoices, with amounts, time for when they were issued and when the demand is due. Link to ledger is displayed.
+
+Payment Demand is the basis for payment claims that the system generates, and it is on the basis of these that an invoice is formed. Examples of using payment requirements k an be reporting outstanding guilty amount or income broken down by month of the year. Each claim points to the subscription period that is the basis for the claim. It is important here to keep in mind that the requirement is formed before the period is formed, so the pointer is therefore always backwards in time.  
+
+For orders, there will be a pointer to the order that led to the claim.
+ 
+
+[economy].[SubscriberAmounts]
+-----------------------------
+Information on all amounts paid for a subscriber, divided into different subscriptions.
+
+ 
+
+[economy].[SubscriberLedgers]
+-----------------------------
+Displays the customer ledger and the types of financial transactions that have been performed for the subscriber.
+
+Special fields in SubscriberLedgers:
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+**SubscriberAccount**: When you create a new subscription, then assigned this one SubscriberAccount. This is retained when the subscription is renewed. But if you have two parallel subscriptions, these will each have their own subscriber account .
+
+**LedgerType** - some examples of values:
+
++---------+------------------------------+
+| Type    | Description                  |
++=========+==============================+
+| Charge  | Payment requirements         |
++---------+------------------------------+
+| Payment | Payment                      |
++---------+------------------------------+
+| Balance | Balance / balance adjustment |
++---------+------------------------------+
+| Credit  | Credit                       |
++---------+------------------------------+
+
+
+
+[Order].[OrderAmounts]
+------------------------
+Displays all orders in total (created, canceled and executed) and the value of these. Formed when an order is created and updated when it is completed or canceled.
+
+ 
+
+[Order].[OrderCompletedAmounts]
+-------------------------------
+Displays the total number of orders executed and the total value of these. The TemplatePackageName field shows which package template was used .
```

Comment:
Grouped by date., templatepackage and organization. Only completed orders.

Reply:
Updated